tests/rptest/wasm/topics_result_set.py

```diff
@@ -31,10 +31,10 @@ class TopicsResultSet:
     A high level wrapper around a result set from a read of a set of kafka
```

Comment:
this commit subject (`rptest/wasm: Sole parameter to topics_result_set was never assigned t…
…o mvar`) overflows on GitHub ui. around 50 characters is a good target.